conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/ConjureJerseyFeature.java

```diff
@@ -16,30 +16,41 @@
 
 package com.palantir.conjure.java.server.jersey;
 
+import com.codahale.metrics.Meter;
 import com.fasterxml.jackson.jaxrs.cbor.JacksonCBORProvider;
 import com.palantir.conjure.java.serialization.ObjectMappers;
 import com.palantir.tracing.jersey.TraceEnrichingFilter;
+import com.palantir.tritium.metrics.registry.DefaultTaggedMetricRegistry;
+import com.palantir.tritium.metrics.registry.TaggedMetricRegistry;
 import javax.ws.rs.core.Feature;
 import javax.ws.rs.core.FeatureContext;
 
 public enum ConjureJerseyFeature implements Feature {
     INSTANCE;
 
+    private final Meter internalErrorMeter;
+
+    ConjureJerseyFeature() {
+        TaggedMetricRegistry registry = new DefaultTaggedMetricRegistry();
```

Comment:
oh hang on this isn't gonna have the outcome you want - metrics only must end up on disk in order to appear in DataDog.  WC makes up a new metric registry and then hooks up sls-logging to make it dump them every 30 seconds. In this case, nothing is hooked up to this instance, so they'll never appear on disk.

I think we _do_ need to use either `DefaultTaggedMetricRegistry.getDefault()` or `SharedTaggedMetricRegistry.getSingleton()` (would prefer the latter as the former was deprecated first).

Reply:
Makes sense. Changed it to use the latter.